data_diff/diff_tables.py

```diff
@@ -231,8 +239,8 @@ def diff_tables(self, table1: TableSegment, table2: TableSegment) -> DiffResult:
         mins, maxs = zip(*key_ranges)
 
         # We add 1 because our ranges are exclusive of the end (like in Python)
-        min_key = min(mins)
-        max_key = max(maxs) + 1
+        min_key = min(map(int, mins))
+        max_key = max(map(int, maxs)) + 1
```

Comment:
This makes sense, what did they pass?

Reply:
Snowflake-connector returned the ids as strings.